metals/src/main/scala/scala/meta/internal/metals/StacktraceAnalyzer.scala

```diff
@@ -206,7 +206,12 @@ object StacktraceAnalyzer {
     val modulePos = symbolIn.indexOf("/")
     val symbolPart =
       if (modulePos > -1) symbolIn.substring(modulePos + 1) else symbolIn
-    val symbol = symbolPart.split('.').init.mkString("/")
+
+    val splitSymbol = symbolPart.split('.')
+    val symbol =
+      if (splitSymbol.size > 1) splitSymbol.init.mkString("/")
+      else splitSymbol.head
```

Comment:
This seems to have always worked under the assumption, that `symbolIn` has a format of `smth.smth.someMethod(...)`, so we drop the last part. But if that isn't always true, shouldn't you check that e.g. the last part contains `(...)` instead of special casing for one element list?

Reply:
No, actually I can't. So where did this bug come from?

Reply:
Ach, actually a good question :thinking: It was happening for me only on windows

Reply:
I will recheck on windows and maybe make the implementation clearer.

Reply:
Ach, so the issue was actually that we search for stacktrace line in something that was not a stacktrace, but had a class name which is why this failed. I changed to always check if `at ` is present